src/GameData/Draw.js

```diff
@@ -0,0 +1,20 @@
+// draw Loop:
+
+function drawBall(canvas, ball) {
+  canvas.fillStyle = ball.style;
+  canvas.fillRect(ball.x, ball.y, ball.w, ball.h);
+}
+
+function drawPaddle(canvas, player) {
+  canvas.fillStyle = player.style;
+  canvas.fillRect(player.x, player.y, player.w, player.h);
+}
+
+export default function drawGameState(canvas, board, ball, player1, player2) {
```

Comment:
The file is called Draw.js while the default function is named drawGameState. If it's a component we should use PascalCase and have the naming match the file name. I'm not sure in your case tho since you further organized your code into subsections